layer.yaml

```diff
@@ -1,31 +1,59 @@
-includes: ['layer:hadoop-client', 'interface:spark', 'interface:benchmark']
+includes:
+- layer:hadoop-client
+- interface:spark
+- interface:benchmark
 options:
-  hadoop-client:
+  apache-spark:
```

Comment:
This isn't how layer options work.  The layer name here is the layer for which you are setting the options, or, in other words, the layer that defines the options you want to set.  In this case, that's `hadoop-client` and setting options for any other layer (`apache-spark`, in this case) will cause them to be ignored (it should probably complain during build instead).